raiden/transfer/utils.py

```diff
@@ -8,7 +8,9 @@ def hash_balance_data(
         locked_amount: typing.TokenAmount,
         locksroot: typing.Locksroot,
 ) -> bytes:
-    if transferred_amount == 0 and locked_amount == 0 and locksroot == b'':
+    if locksroot == b'':
+        locksroot = bytes(32)
```

Comment:
we should stick to one of the representation here, what about defining a constant for the empty locks root and using it instead of accepting multiple inputs?

Reply:
yep I agree, I'll update it also in other places where `b''` is used as a hash.